test/functional/ui/screen_basic_spec.lua

```diff
@@ -73,33 +73,29 @@ describe('Screen', function()
   describe(':suspend', function()
     it('is forwarded to the UI', function()
       local function check()
-        if not screen.suspended then
-          return 'Screen was not suspended'
-        end
+        eq(true, screen.suspended)
       end
```

Comment:
So the only issue is that `screen:expect()` loses the distinction between errors and failures? It does not turn failures or errors into success, and you can still see the full assertion failure message in the output, so I don't really see any blocking problem here.